Project.toml

```
@@ -1,7 +1,7 @@
 name = "CFITSIO"
 uuid = "3b1b4be9-1499-4b22-8d78-7db3344d1961"
 authors = ["Miles Lucas <mdlucas@hawaii.edu> and contributors"]
-version = "1.0.0"
+version = "1.0.1"
```

I think this can be considered a new feature, so I'd bump the minor version:
```suggestion
version = "1.1.0"
```